andela/pippin-ah-backend
# [Feature: #162727556] Create New Article

#### What does this PR do?
This adds functionality for creating an article

#### Description of Task to be completed?
- User should be able to create an article

#### How should this be manually tested?
- Run migration
- Send a POST request to `/api/v1/articles` with required fields
- You should get a 201 success response with your new article

#### What are the relevant pivotal tracker stories?
[#162727556](https://www.pivotaltracker.com/story/show/162727556)
